File: actions/php/setup.ejs.php
require_once('lib/KalturaClient.php');
$config = new KalturaConfiguration(<%- Lucy.variable('answers.partnerID') %>);
$config->serviceUrl = 'https://www.kaltura.com/';
$KalturaClient = new KalturaClient($config);
$KalturaSession = $KalturaClient->session->start(
    <%- Lucy.variable('answers.adminSecret') %>,
    "myUser",
    KalturaSessionType::ADMIN,
    <%- Lucy.variable('answers.partnerID') %>,
    null, null);
$KalturaClient->setKS($KalturaSession);

File: actions/ruby/setup.ejs.rb
require "ruby_client.rb"
include Kaltura
 
kalturaConfig = KalturaConfiguration.new(<%- Lucy.variable('answers.partnerID') %>)
kalturaConfig.service_url = 'http://www.kaltura.com/'
kalturaClient = KalturaClient.new(kalturaConfig)
kalturaSession = client.session_service.start(
    <%- Lucy.variable('answers.adminSecret') %>,
    userId,
    type,
    partnerId,
    expiry, privileges)
kalturaClient.ks = kalturaSession
